govcd/adminorg_test.go

```diff
@@ -0,0 +1,38 @@
+// +build org functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+package govcd
+
+import (
+	"fmt"
+	. "gopkg.in/check.v1"
+)
+
+// Creates a Catalog and then verify that finds it
+func (vcd *TestVCD) Test_Client_FinAdminCatalogByName(check *C) {
```

Comment:
Change function name to reflect suggestion about main function above

Reply:
fixed